packages/fxa-event-broker/bin/simulate-webhook-call.ts

```diff
@@ -36,6 +36,7 @@ async function main() {
   const uid = chance.hash();
   const jwtPayload = await jwt.generateSubscriptionSET({
     capabilities: args.capabilities.split(','),
+    changeTime: Math.trunc(Date.now() / 1000),
```

Comment:
The docs above say `changeTime` is time in milliseconds, but here you divide by 1000, converting to seconds.

Reply:
The seconds value is correct, changing the docs.